backend/api/Controllers/MissionController.cs

```diff
@@ -66,10 +75,57 @@ IStidService stidService
             return BadRequest("Max EndTime cannot be less than min EndTime");
         }
 
-        PagedList<Mission> missions;
+        PagedList<MissionRun> missions;
+        try
+        {
+            missions = await _missionRunService.ReadAll(parameters);
+        }
+        catch (InvalidDataException e)
+        {
+            _logger.LogError(e.Message);
+            return BadRequest(e.Message);
+        }
+
+        var metadata = new
+        {
+            missions.TotalCount,
+            missions.PageSize,
+            missions.CurrentPage,
+            missions.TotalPages,
+            missions.HasNext,
+            missions.HasPrevious
+        };
+
+        Response.Headers.Add(
+            QueryStringParameters.PaginationHeader,
+            JsonSerializer.Serialize(metadata)
+        );
+
+        return Ok(missions);
+    }
+
+    /// <summary>
+    /// List all mission definitions in the Flotilla database
+    /// </summary>
+    /// <remarks>
+    /// <para> This query gets all missions </para>
+    /// </remarks>
+    [HttpGet("definitions")]
+    [Authorize(Roles = Role.Any)]
+    [ProducesResponseType(typeof(IList<MissionDefinition>), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public async Task<ActionResult<IList<MissionDefinition>>> GetMissionDefinitions(
+        [FromQuery] MissionDefinitionQueryStringParameters parameters
+    )
+    {
+        // TODO: define new parameters using primarily area and source type
```

Comment:
```suggestion
```
Add this as a separated issue if it will not be implemented in this PR

Reply:
This PR is big enough as it is so please do a new issue and a new PR later.

Reply:
https://github.com/equinor/flotilla/issues/878